faucetagent.py

```diff
@@ -369,8 +371,10 @@ def main():
         dp_wait_fraction=args.dpwait,
         timeout=args.timeout,
         nohup=args.nohup)
-    # FaucetAgent handles gNMI requests
+
+    # FaucetAgent handles gNMI requestsn
```

Comment:
tailing 'n' here that should be removed.

Reply:
emacs key bindings ftl